Jenkinsfile

```diff
@@ -78,8 +78,11 @@ pipeline {
         stage('Azure Authenticator preparation - Allocate Azure Authenticator Instance') {
```

Comment:
I think "Allocate Azure Authenticator Instance" is enough.

Reply:
because this stage runs in parallel with all the other tests (and not only azure) i wanted to make it obvious that it's related to the azure authenticator tests